scripts/test/js_test.sh

```diff
@@ -31,8 +28,8 @@ if [[ ! -z "$2" ]]; then
     CMD_ARGS+=" -g \"$2\""
 fi
 
-echo "Running: $CMD $CMD_ARGS"
-eval "$CMD $CMD_ARGS" 2> >(tee "$TMP_FILE")
+echo "Running: $CMD"
+eval "$CMD" 2> >(tee "$TMP_FILE")
```

Comment:
Is it not possible to target test cases by name with Jest? That's an unfortunate drawback. If that's true, we should also get rid of the section above that parses the extra command args (line 17-29) since we're not doing anything with it

Reply:
oh probably it is - I'll look into that. I also sort of eviscerated the `js_test.sh` file here and forgot about coverage too - I'll see about making that work with it

Reply:
ok, so a question for you about this. I changed it over to run jest directly in `package.json` scripts, rather than using this `js_test.sh` script because it was causing the test output to be printed twice.

so running `npm test` will run `jest`, `npm run test:watch` will run `jest --watch`, and so on.

with this setup, you can run a test by matching on its name by doing `npm run -- -t "test name"`. we only have one test right now :joy: but I tried it and it seems to work to me.

does that work for your use-case for specifying test names?

Reply:
also adjusted the coverage setup and I think got that working.

Reply:
@aliceriot That sounds great. Could you update the README to include these details? (the new command for running tests, "watching", running test cases by description, etc.)

Reply:
definitely!